lib/cmd/common/deploy-target.js

```diff
@@ -20,7 +20,6 @@ deployTargets.usage = "\nfhc deploy-target create name=<the name of the deploy t
                     + "\nfhc deploy-target setapptarget <appid> <id of the deploy target> --env=<environment>"
```

Comment:
@wtrocki Tks for the help here. The common has the commands that we are migrating for the V3 standard. It is not all migrated yet and we are doing it command by command. So, it will take a few time to disappear but the idea will that will be out either and as soon as possible :-)